src/handlers/assign.rs

```diff
@@ -90,13 +90,19 @@ Please choose another assignee.";
 // Special account that we use to prevent assignment.
 const GHOST_ACCOUNT: &str = "ghost";
 
+/// Assignment data stored in the issue/PR body.
 #[derive(Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 struct AssignData {
     user: Option<String>,
 }
 
-/// Input for auto-assignment when a PR is created.
-pub(super) struct AssignInput {}
+/// Input for auto-assignment when a PR is created or converted from draft.
+#[derive(Debug)]
+pub(super) enum AssignInput {
+    Opened,
+    OpenedDraft,
```

Comment:
nit: I think this would be better expressed as `Opened(bool)` with the boolean being the draft status